pricing page for workflow (#251)
upstash/docs: qstash/workflow/pricing.mdx

---
title: "Pricing"
---

> Upstash Workflows are built on QStash, and the pricing model follows a "pay-as-you-go" approach. You only pay when your app receives traffic, so there's no cost when it's not in use.

Reviewer: ```suggestion
Upstash Workflow is built on QStash, and the pricing model follows a "pay-as-you-go" approach. You only pay when your app receives traffic, so there's no cost when it's not in use.
```

Reviewer: We decided to use `Upstash Workflow` in singular form.

> Each workflow run consists of multiple QStash messages, with the total cost determined by the number of messages used.

Reviewer: ```suggestion
A workflow run consists of several QStash messages, with the total cost determined by the number of messages used.
```
